ftplugin/javascript.vim

```diff
@@ -1,4 +1,4 @@
 setlocal suffixesadd+=.js
-if v:version > 704 || (v:version == 704 && has('patch002'))
+if (v:version < 704 || (v:version == 704 && has('patch002'))) && exists('&regexpengine')
```

Comment:
We need a NOT on the `has()` patch, since patch fixes, and therefore, we don't want `re=1` on `has()` being truthy.